Source/WebCore/html/HTMLIFrameElement.cpp

```diff
@@ -252,6 +254,21 @@ LazyLoadFrameObserver& HTMLIFrameElement::lazyLoadFrameObserver()
     return *m_lazyLoadFrameObserver;
 }
 
+void HTMLIFrameElement::resetPermissionsPolicyDirective()
+{
+    m_permissionsPolicyDirective = std::nullopt;
+
+    RefPtr remoteContentFrame = dynamicDowncast<RemoteFrame>(contentFrame());
+    if (!remoteContentFrame)
+        return;
+
+    WebCore::OwnerPermissionsPolicyData policy;
```

Comment:
```suggestion
    OwnerPermissionsPolicyData policy;
```
Also, why not initializing the members directly in constructor?